packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -167,7 +167,7 @@ export const sortings: Record<PostSortingMode,MongoSelector<DbPost>> = {
  * as it is *inclusive*. The parameters callback that handles it outputs
  * ~ $lt: before.endOf('day').
  */
-Posts.addDefaultView((terms: PostsViewTerms, _, context: ResolverContext) => {
+Posts.addDefaultView((terms: PostsViewTerms, _, context?: ResolverContext) => {
```

Comment:
Did we discover that context is optional here?

Reply:
I just tried removing the diff and didn't see any errors, but it was added by @jimrandomh I suspect he had some kind of reason.